commit da1d9caf95def6f0320819cf941c9fd1069ba9e1
Author: Federico Vaga <federico.vaga@vaga.pv.it>
Date:   Sat Jul 2 23:08:20 2022 +0200

    doc:it_IT: align Italian documentation
    
    Translation for the following patches
    
    commit df05c0e9496c ("Documentation: Raise the minimum supported version of LLVM to 11.0.0")
    commit 333b11e541fe ("Documentation: Add minimum pahole version")
    commit 6d6a8d6a4ed0 ("docs: Update Sphinx requirements")
    commit 76ae847497bc ("Documentation: raise minimum supported version of GCC to 5.1")
    commit 59c6a716b14b ("Documentation/process/maintainer-pgp-guide: Replace broken link to PGP path finder")
    commit 85eafc63d032 ("docs: update file link location")
    commit 869f496e1aa6 ("docs: process: submitting-patches: Clarify the Reported-by usage")
    commit 6c5ccd24ff17 ("Remove mentions of the Trivial Patch Monkey")
    commit aa9b5e0df226 ("Documentation/process: fix self reference")
    commit b96ff02ab2be ("Documentation/process: fix a cross reference")
    commit 1f57bd42b77c ("docs: submitting-patches: make section about the Link: tag more explicit")
    commit a9d85efb25fb ("docs: use the lore redirector everywhere")
    commit 31c9d7c82975 ("Documentation/process: Add tip tree handbook")
    commit 604370e106cc ("Documentation/process: Add maintainer handbooks section")
    commit bf33a9d42d0c ("docs: 5.Posting.rst: describe Fixes: and Link: tags")
    commit c04639a7d2fb ("coding-style.rst: trivial: fix location of driver model macros")
    commit d5b421fe0282 ("docs: Explain the desired position of function attributes")
    commit 3577cdb23b8f ("docs: deprecated.rst: Clarify open-coded arithmetic with literals")
    commit db67eb748e7a ("docs: discourage use of list tables")
    commit 0e805b118662 ("docs: address some text issues with css/theme support")
    commit 135707d3765e ("docs: allow to pass extra DOCS_CSS themes via make")
    commit fe450eeb4e6f ("Documentation: in_irq() cleanup")
    commit 10855b45a428 ("docs: fix typo in Documentation/kernel-hacking/locking.rst")
    commit bc67f1c454fb ("docs: futex: Fix kernel-doc references")
    commit abf36fe0be7d ("docs: kernel-hacking: Remove inappropriate text")
    commit f35cf1a59e9a ("Documentation: kernel-hacking: minor edits for style")
    commit f35cf1a59e9a ("Documentation: kernel-hacking: minor edits for style")
    commit 980c3799c500 ("Documentation: kernel-doc: Promote two chapter headings to page title")
    commit e1be43d9b5d0 ("overflow: Implement size_t saturating arithmetic helpers")
    commit 615f3eea0d5f ("Documentation: add note block surrounding security patch note")
    commit 587d39b260c4 ("Documentation: add link to stable release candidate tree")
    commit 555d44932c67 ("Documentation: update stable tree link")
    commit 88d99e870143 ("Documentation: update stable review cycle documentation")
    commit 0c603a5c704f ("Documentation/process: mention patch changelog in review process")
    commit 6d5aa418b3bd ("docs: submitting-patches: Fix crossref to 'The canonical patch format'")
    commit f1a693994b1c ("Documentation/process: use scripts/get_maintainer.pl on patches")
    commit 69ef0920bdd3 ("Docs: Add cpio requirement to changes.rst")
    commit 5a5866c28b43 ("Docs: Replace version by 'current' in changes.rst")
    commit 9b5a7f4a2a8d ("x86/configs: Add x86 debugging Kconfig fragment plus docs")
    commit f1a693994b1c ("Documentation/process: use scripts/get_maintainer.pl on patches")
    commit e8c07082a810 ("Kbuild: move to -std=gnu11")
    
    Signed-off-by: Federico Vaga <federico.vaga@vaga.pv.it>
    Link: https://lore.kernel.org/r/20220702210820.13118-1-federico.vaga@vaga.pv.it
    Signed-off-by: Jonathan Corbet <corbet@lwn.net>

diff --git a/Documentation/translations/it_IT/process/changes.rst b/Documentation/translations/it_IT/process/changes.rst
index dc7193377b7f..10e0ef9c34b7 100644
--- a/Documentation/translations/it_IT/process/changes.rst
+++ b/Documentation/translations/it_IT/process/changes.rst
@@ -11,8 +11,8 @@ Requisiti minimi per compilare il kernel
 Introduzione
 ============
 
-Questo documento fornisce una lista dei software necessari per eseguire i
-kernel 4.x.
+Questo documento fornisce una lista dei software necessari per eseguire questa
+versione del kernel.
 
 Questo documento è basato sul file "Changes" del kernel 2.0.x e quindi le
 persone che lo scrissero meritano credito (Jared Mauch, Axel Boldt,
@@ -32,12 +32,13 @@ PC Card, per esempio, probabilmente non dovreste preoccuparvi di pcmciautils.
 ====================== =================  ========================================
         Programma       Versione minima       Comando per verificare la versione
 ====================== =================  ========================================
-GNU C                  4.9                gcc --version
-Clang/LLVM (optional)  10.0.1             clang --version
+GNU C                  5.1                gcc --version
+Clang/LLVM (optional)  11.0.0             clang --version
 GNU make               3.81               make --version
 binutils               2.23               ld -v
 flex                   2.5.35             flex --version
 bison                  2.0                bison --version
+pahole                 1.16               pahole --version
 util-linux             2.10o              fdformat --version
 kmod                   13                 depmod -V
 e2fsprogs              1.41.4             e2fsck -V
@@ -58,6 +59,7 @@ iptables               1.4.2              iptables -V
 openssl & libcrypto    1.0.0              openssl version
 bc                     1.06.95            bc --version
 Sphinx\ [#f1]_         1.7                sphinx-build --version
+cpio                   any                cpio --version
 ====================== =================  ========================================
 
 .. [#f1] Sphinx è necessario solo per produrre la documentazione del Kernel
@@ -111,6 +113,16 @@ Bison
 Dalla versione 4.16, il sistema di compilazione, durante l'esecuzione, genera
 un parsificatore.  Questo richiede bison 2.0 o successivo.
 
+pahole
+------
+
+Dalla versione 5.2, quando viene impostato CONFIG_DEBUG_INFO_BTF, il sistema di
+compilazione genera BTF (BPF Type Format) a partire da DWARF per vmlinux. Più
+tardi anche per i moduli. Questo richiede pahole v1.16 o successivo.
+
+A seconda della distribuzione, lo si può trovare nei pacchetti 'dwarves' o
+'pahole'. Oppure lo si può trovare qui: https://fedorapeople.org/~acme/dwarves/.
+
 Perl
 ----
 
@@ -455,6 +467,11 @@ mcelog
 
 - <http://www.mcelog.org/>
 
+cpio
+----
+
+- <https://www.gnu.org/software/cpio/>
+
 Rete
 ****